include/xrpl/protocol/TxFlags.h

```diff
@@ -58,8 +58,10 @@ namespace ripple {
 // clang-format off
 // Universal Transaction flags:
 constexpr std::uint32_t tfFullyCanonicalSig                = 0x80000000;
+constexpr std::uint32_t tfInnerBatchTxn                         = 0x40000000;
 constexpr std::uint32_t tfUniversal                        = tfFullyCanonicalSig;
-constexpr std::uint32_t tfUniversalMask                    = ~tfUniversal;
+constexpr std::uint32_t tfUniversalV2                      = tfFullyCanonicalSig | tfInnerBatchTxn;
+constexpr std::uint32_t tfUniversalMask                    = ~(tfFullyCanonicalSig | tfInnerBatchTxn);
```

Comment:
```suggestion
constexpr std::uint32_t tfUniversal = tfFullyCanonicalSig | tfInnerBatchTxn;
constexpr std::uint32_t tfUniversalMask                    = ~(tfFullyCanonicalSig | tfInnerBatchTxn);
```

we can prob add `tfInnerBatchTxn` into `tfUniversal` directly, and check in `Transactor::preflight1` if `tfInnerBatchTxn` is usable